cms/djangoapps/contentstore/xblock_storage_handlers/__init__.py

```diff
@@ -3,4 +3,4 @@
 """
 from .create_xblock import *
 from .xblock_helpers import *
-from .xblock_service import *
+from .view_handlers import *
```

Comment:
Hm, purely based on personal aesthetic, I think would call these `xblock_operations`, as "handlers" often refer to callbacks which respond to (ie, handle) an event or a request.

That said, unlike "XBlock Serivces", there is no existing term called "XBlock Handlers" so I see no real risk for confusion. If `xblock_storage_handlers` feels right, then that is OK with me 👍🏻

Reply:
Difficult decision. I feel the `xblock_storage_handlers` name is more correct. I'm not hundred percent certain about `view_handlers.py`, but I'd leave it, as it seems good enough as a name, and I can't really invest in even more refactoring right now.